src/client.cpp

```diff
@@ -2199,13 +2199,13 @@ void Client::sendChangePassword(const std::wstring &oldpassword,
 }
 
 
-void Client::sendDamage(u8 damage)
+void Client::sendDamage(s8 damage)
 {
 	DSTACK(__FUNCTION_NAME);
 	std::ostringstream os(std::ios_base::binary);
 
 	writeU16(os, TOSERVER_DAMAGE);
-	writeU8(os, damage);
+	writeS8(os, damage);
```

Comment:
idk, if you use s8 you can't let something make > 78 damage per second, can you?